### ISSUE TITLE
Feature request: Admin UI - hide/censor content - field option

### ISSUE THREAD

**Issue by peter-pakanun** [pocketbase/pocketbase#4760]
It would be nice to be able to hide text content (plaintext, email, etc.) from showing up in Admin UI, something like this:
![image](https://github.com/pocketbase/pocketbase/assets/6493474/cf2d489a-0af1-46c7-b0fa-649a126936cd)
And maybe blur the textarea using css like this:
![image](https://github.com/pocketbase/pocketbase/assets/6493474/d207ab82-7fa1-4704-b988-9ed62eea522d)

From what I understand, the `hiddenColumns` is stored in client localstorage, this always made me paranoid because I use a lots of difference browsers and/or profile. Often times I find myself have to grab the browser window to other monitor before accessing the Admin UI and then double check to see if the sensitive columns was hidden or not before I can move it back for my viewer to see. This is prone to human error and I think I'd make a mistake someday and leak sensitive data.

This is mainly a streamer thing, but it also apply to situation where you have to let someone else unrelated to the project see the structure of the data without the sensitive content, maybe in-person or remotely via screenshare (like asking someone from other team—who is standing behind you—for db advice) in this case hiding the columns also defeat the purpose.

**Comment by ganigeorgiev** [comment]
`hidden` field option that will prevent a field to be returned in the API response is already implemented as part of https://github.com/pocketbase/pocketbase/discussions/4355.

**Comment by peter-pakanun** [comment]
> `hidden` field option that will prevent a field to be returned in the API response...

Will the user still be able to read the hidden content in this case?

What I mean by hiding sensitive content is that the content itself is not hidden to anyone with API access to it, if they list it it should still return from the API, but it should be hidden/censored in the Admin UI, just like what input type password would do.

**Comment by ganigeorgiev** [comment]
No, the `hidden` option will exclude the field from the serialization and filters.

There are no plans for now for just visually "hide" the field content as I don't think the use case is very common.

**Comment by pauserratgutierrez** [comment]
> No, the `hidden` option will exclude the field from the serialization and filters.
> 
> There are no plans for now for just visually "hide" the field content as I don't think the use case is very common.

Hi Gani,

It's true that only in specific cases someone would like to hide a field from the UI, but I personally faced the necessity of doing it in multiple Pocketbase projects of mine.

I needed to have a field to display the duration of a service. I'm using two fields: **Field 1:** For the user to enter a duration in a more friendly way, by simply doing
`00h 00m 00s`
![image](https://github.com/user-attachments/assets/567a12db-fa42-4e7c-9928-e1ee799eac7e)

**Field 2:** For the system to save a normalized version of the previous input, being a duration converted to seconds (a custom hook detects a creation or update of a record and takes the Field 1 and automatically does a conversion to seconds, and stores it in Field 2)
![image](https://github.com/user-attachments/assets/c036c471-6200-446a-9b65-6be1d7f98679)

In this specific case, Field 1 is what the user uses to enter the data, but Field 2 isn't used by users, it's a system thing, so it shouldn't show in the UI, and even should have a way to prevent someone from changing it.

I have another usecase, where I use a text input for the user to enter a location string. A custom hook does an API call to nominatum, retrieves the exact location for that specific text and saves the location into a specific field, that is something handled only by the system.

I just wanted to open a new feature request, but after doing a first search I found this. If you want me to open a new issue, I will be more than happy to do it!
